Applications/Games/Batman™: Arkham Origins/Steam/script.js

```diff
@@ -0,0 +1,16 @@
+include(["Engines", "Wine", "QuickScript", "SteamScript"]);
+
+new SteamScript()
+    .name("Batman™: Arkham Origins")              
+    .editor("WB Games Montreal, Splash Damage")     
+    .author("ImperatorS79")
+    .wineVersion("2.15")
```

Comment:
If it runs with Wine 1.7 already, I'd rather go for the default Wine version.

Reply:
I have chosen 2.15-staging because the game has a DirectX11 mode

Reply:
Ok.